test/setupTests.ts

```diff
@@ -38,9 +38,11 @@ beforeEach(() => {
 });
 
 // Retry to work around our flaky app & tests
-jest.retryTimes(2, {
-    logErrorsBeforeRetry: true,
-});
+if (process.env.CI) {
```

Comment:
I'm thinking this should be in a separate PR

Reply:
Sure

Reply:
[Done](https://github.com/matrix-org/matrix-react-sdk/pull/12111)